app/services/prompt_augmentor.rb

```diff
@@ -12,21 +12,24 @@ def augment
 
   def prompt
     @prompt ||= if @search_hits.any?
-      prompt = "You are given a question to answer based on some given textual context, all inside xml tags.\nIf the answer is not in the context but you think you know the answer, explain that to the user then answer with your own knowledge.\n\n"
+      prompt = "You are given a question to answer based on some given textual context, all inside xml tags.\nIf the answer is not in the context but you think you know the answer, explain that to the user then answer with your own knowledge.\n\n" # rubocop:disable Layout/LineLength
```

Comment:
Converting to a HEREDOC would let you avoid the `rubocop:disable`.

Reply:
I might do that, but I'm finding it easier to do this way for now.